src/uct/sm/scopy/knem/knem_md.c

```diff
@@ -203,6 +203,7 @@ ucs_status_t uct_knem_md_query(uct_md_h uct_md, uct_md_attr_v2_t *md_attr)
         md_attr->flags         |= UCT_MD_FLAG_REG;
         md_attr->reg_mem_types |= UCS_BIT(UCS_MEMORY_TYPE_HOST);
     }
+    md_attr->reg_nonblock_mem_types = 0;
```

Comment:
why is this needed?

Reply:
I just discovered that during this query function we do initialize `reg_mem_types` but do not initialize `reg_nonblock_mem_types` so I decided to add the initialization.